modules/op-release-notes-1-13.adoc

```diff
@@ -0,0 +1,131 @@
+// This module is included in the following assemblies:
+// * about/op-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="op-release-notes-1-13_{context}"]
+= Release notes for {pipelines-title} General Availability 1.13
+
+With this update, {pipelines-title} General Availability (GA) 1.13 is available on {OCP} 4.12 and later versions.
+
+[id="new-features-1-13_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.13:
+
+[NOTE]
+====
+Before upgrading to the {pipelines-title} Operator 1.13, ensure that you have at least installed the {OCP} 4.12.19 or 4.13.1 version on your cluster.
+====
+
+[id="pipelines-new-features-1-13_{context}"]
+=== Pipelines
+
+* Before this update, the Source-to-Image (S2I) cluster tasks used a base S2I container image that was in Technology Preview. With this update, the S2I cluster tasks used a base S2I container image that is released and fully supported.
+
+* With this update, optionally, you can enable a setting so that OpenShift Pipelines stops the pod for the task run but does not delete the pod. To enable this setting, set the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to `true` and the `pipeline.enable-api-fields` spec to `alpha` in the `TektonConfig` custom resource (CR).
```

Comment:
@VeereshAradhya I see a discussion on this, Still these points are valid?
https://redhat-internal.slack.com/archives/C03A1MBHN81/p1701098782497289

Reply:
Yes. We need to add known issues related to this

Reply:
what are known issues related to these are they already covered @VeereshAradhya ?